torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -460,14 +462,22 @@ void IndexCompute::handle(Merge* merge) {
     // Shouldn't hit this, but don't want to segfault if somehow we do.
     TORCH_INTERNAL_ASSERT(!input_ids.empty());
 
+    kir::Val* extent = nullptr;
```

Comment:
Is this variable necessary? It's set but doesn't seem to be used.

Reply:
Yeah, I think I can actually revert most of this, this was left over from the old approach.